pygmt/datasets/earth_relief.py

```diff
@@ -144,12 +144,13 @@ def load_earth_relief(
     earth_relief_sources = {
         "igpp": "earth_relief_",
         "gebco": "earth_gebco_",
+        "gebcosi": "earth_gebcosi_",
         "synbath": "earth_synbath_",
```

Comment:
Need to update the `data_source` parameter docstring at L71-L81 about the new `gebcosi` (and `synbath`) options.

Reply:
Added!